docs/Upgrade.md

```diff
@@ -0,0 +1,38 @@
+# Splunk Operator Upgrade
+
+Please note upgrading Splunk operator to Version `1.1.0` is a new installation rather than upgrade from current operator. Due to this  older Splunk operator need to be cleaned up before installing the latest version.  The script will help customer in doing these steps. This script expect current namespace where operator is installed , and path to `1.1.0` manifest file. script will follow the below mentioned steps
+
+* Backup of all the operator resources within the namespace like
+** service-account, deployment, role, role-binding, cluster-role, cluster-role-binding
+* Deletes all the old Splunk operator resources and deployment
+* Installs the operator 1.1.0 in Splunk-operator namespace.
+
+By default Splunk operator 1.1.0 will be installed to watch cluster-wide
+
+## Steps for upgrade from 1.0.5 to 1.1.0
+
+run upgrade-to-1.1.0.sh script with below mentioned mandatory arguments
```

Comment:
change "below mentioned mandatory arguments" to "the following mandatory arguments"

Reply:
Fixed